src/textual/reactive.py

```diff
@@ -28,12 +26,6 @@
 ReactiveType = TypeVar("ReactiveType")
 
 
-class _NotSet:
```

Comment:
this is unused. I assume this once was thought of as a possibility for detecting "first time", or "default", but the impl has changed since, and there is no such thing anymore as "uninitialized".